src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int ARRAY_SIZE = 10;
+    private static final double INCREASE_ARRAY = 1.5;
+    private Object[] arraylist = new Object[ARRAY_SIZE];
```

Comment:
please initialise `arraylist` in the constructor

Reply:
Done;